.env.example

```diff
@@ -189,3 +189,11 @@ INTERNET_COMPUTER_ADDRESS=
 # Aptos
 APTOS_PRIVATE_KEY= # Aptos private key
 APTOS_NETWORK=     # must be one of mainnet, testnet
+
+# Charity Configuration
+IS_CHARITABLE=false   # Set to true to enable charity donations
+CHARITY_ADDRESS_BASE=0x1234567890123456789012345678901234567890
+CHARITY_ADDRESS_SOL=pWvDXKu6CpbKKvKQkZvDA66hgsTB6X2AgFxksYogHLV
+CHARITY_ADDRESS_ETH=0x750EF1D7a0b4Ab1c97B7A623D7917CcEb5ea779C
+CHARITY_ADDRESS_ARB=0x1234567890123456789012345678901234567890
+CHARITY_ADDRESS_POL=0x1234567890123456789012345678901234567890
```

Comment:
Can you please update the BASE address to the ETH address they should be the same.

Reply:
And arb and poly should be empty for now as they are dummy addresses atm :)

Reply:
we're removing hardcoded wallets its weird to force projects to give charity